style.css

```diff
@@ -0,0 +1,959 @@
+  margin: 3% 0%;
+  line-height: 2rem;
+  font-size: 2rem;
+  font-family: 'Noto Sans', sans-serif;
+  color: #ec5242;
+}
+
+#aboutDetail {
+  background-color: white;
+  color: #535353;
+  font-size: 0.85rem;
+  border: 1.5px solid rgb(236, 236, 236);
+  padding: 3%;
+  margin: 6% 0% 2% 0%;
+  text-align: center;
+  line-height: 1.5rem;
+}
+
+#aboutDetail p {
+  margin: 0;
+}
+
+@media (max-width: 767px) {
+  .deskp {
+    display: none;
+  }
+
+  #mobilepart {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #desktop {
+    display: block;
+  }
+
+  #mobilepart {
+    display: block;
+  }
+
+  #topnav {
+    display: block;
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    align-items: center;
+    width: 100%;
+    background-color: rgb(60, 62, 66);
+    gap: 2%;
+  }
+
+  .programs img {
+    width: 20%;
+    object-fit: scale-down;
+    margin-left: 40%;
+    margin-top: 10%;
+  }
+
+  .sponsors img {
+    width: 150px;
+  }
+
+  #topnav img {
+    max-height: 18px;
+  }
+
+  #topnav a {
+    margin-right: 2%;
+  }
+
+  #topnavList {
+    margin: 0%;
+    padding: 1%;
+    list-style: none;
+    display: inline-flex;
+    justify-content: space-evenly;
+    gap: 15%;
+    padding-right: 10%;
+  }
+
+  #topnavList li {
+    font-size: 1.2rem;
+    color: rgb(255, 255, 255);
+  }
+
+  .social {
+    max-height: 20px;
+    max-width: 10rem;
+    color: rgb(255, 255, 255);
+  }
+
+  .dropDown {
+    display: flex;
+    background-color: #fff;
+  }
+
+  .mlogo1 {
+    display: block;
+    text-decoration: none;
+    font-size: 40px;
+    font-weight: 900;
+    color: #742820;
+    cursor: pointer;
+    margin-left: 80px;
+    align-self: center;
+  }
+
+  #menuOpen {
+    display: none;
+  }
+
+  #menuList {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+    gap: 2%;
+    align-items: center;
+    list-style: none;
+    padding: 0%;
+    margin: 0%;
+    width: 100%;
+  }
+
+  #menuList a {
+    font-size: 1.2rem;
+    color: #272a31;
+  }
+
+  .campaignBtn {
+    justify-content: end;
+    padding: 1vw;
+    color: #ec5242;
+    background-color: #fff;
+    font-weight: 500;
+    font-size: 15px;
+    margin-left: 40px;
+    border: 5px solid #ec5242;
+    margin-top: 10px;
+    margin-bottom: 5px;
+    margin-right: 8vw;
+  }
+
+  #intro {
+    background-image: url(./Assets/desktopShapes.jpg);
+    padding-top: 10%;
+    padding-left: 15%;
+    padding-right: 25%;
+  }
+
+  #hello {
+    font-size: 4rem;
+  }
+
+  .deskhello {
+    display: none;
+  }
+
+  #autoFest {
+    font-size: 5rem;
+    line-height: 6rem;
+  }
+
+  #eventDetail {
+    font-size: 1.2rem;
+    width: 70%;
+  }
+
+  #eventDetail p {
+    line-height: 2rem;
+  }
+
+  #date {
+    font-size: 2.5rem;
+  }
+
+  #location {
+    font-size: 1.5rem;
+  }
+
+  #mainProgram {
+    font-size: 3rem;
+  }
+
+  #programContainer {
+    display: flex;
+    gap: 3px;
+    width: 90%;
+  }
```

Comment:
- [ ] Kindly consider styling the main container of your program using `grid` instead of `flex` as it will enhance responsiveness across all screen sizes. The `grid` layout provides more fine-grained control over the positioning and alignment of elements, making it easier to achieve responsive designs. 
   - [ ] To refactor your code to use `grid` you can add this style:  `display: grid`, `grid-template-columns: repeat(5, 1fr)` these styles would make sure that each of your program cards has `1` fraction of the spacing available no matter the screen size.
   - [ ] On screen sizes between `768px` and `1000px` consider displaying the cards `3` on a row and `2` below it so they can all have enough space and help it look stunning to create a good user experience. See below for more details: 
   
<img width="782" alt="Screenshot 2023-07-07 at 00 16 56" src="https://github.com/Godedict/Auto-Conference/assets/107192076/ad7c9dfa-5b3d-4b35-a3bf-c1d5ff9f81bc">

Reply:
done